hikari/impl/cache.py

```diff
@@ -1459,12 +1455,13 @@ def _garbage_collect_message(
         *,
         decrement: typing.Optional[int] = None,
         override_ref: bool = False,
-    ) -> typing.Optional[cache_utility.RefCell[cache_utility.MessageData]]:
+    ) -> bool:
```

Comment:
This change seems unnecessary and breaks the standard pattern for these methods

Reply:
The other methods currently only return `None`. I changed this since I dont think it makes sense to return the exact same thing that was sent in, considering how we are using this function atm

Reply:
The methods which were previously only returning None are doing this because they have no relevant delete and clear methods, they are not the same case

Reply:
> think it makes sense to return the exact same thing that was sent in,

Look at how clear members was using it before